### PR TITLE
Support for React Native 0.29+

### LINKED ISSUE #5: RN0.29+
Hello, first of all I would like to thank you!! It's an awesome module =D But I was wondering if you have any plans to release a new version which is compatible with RN0.29+?

Again, thank you!

### PR DESCRIPTION
**Pull request by alwx**
Adds support for React Native 0.29, fixes #5

### CODE REVIEW

**Comment by mCodex** [comment]
Thank you, man! 😆

File: android/build.gradle [hunk 1]
```diff
@@ -14,5 +14,5 @@ android {
 
 dependencies {
     compile 'com.android.support:appcompat-v7:23.1.1'
-    compile 'com.facebook.react:react-native:0.20.+'
+    compile 'com.facebook.react:react-native:0.19.+'
```
Comment (andreyvital) [inline android/build.gradle]: `19`?

**Comment by alwx** [comment]
Updated RN version

**Comment by andreyvital** [comment]
@mCodex published in npm, 0.3.0.

**Pull request merged**